internal/manager/config_validation.go

```diff
@@ -105,6 +109,16 @@ func (c *Config) validateKongAdminAPI() error {
 	return nil
 }
 
+func (c *Config) validateFallbackConfiguration() error {
+	if !c.FeatureGates[featuregates.FallbackConfiguration] && c.UseLastValidConfigForFallback {
+		return fmt.Errorf(
+			"--use-last-valid-config-for-fallback or USE_LAST_VALID_CONFIG_FOR_FALLBACK can only be used with %s feature gate enabled",
```

Comment:
Do we have those env vars `USE_LAST_VALID_CONFIG_FOR_FALLBACK` without any prefix?